cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -93,13 +93,24 @@ func NewDefaultTestServerOptions() *TestServerInstanceOptions {
 	}
 }
 
-// StartTestServer starts a etcd server and kube-apiserver. A rest client config and a tear-down func,
+// StartTestServerOrDie calls StartTestServer t.Fatal if it does not succeed.
```

Comment:
The diff looks a little confusing, but all I did here is make the StartTestServer function private.